lib/core/ArgumentParser.py

```diff
@@ -58,10 +58,13 @@ def __init__(self, script_path):
         else:
             self.urlList = [options.url]
 
-        if options.extensions == None:
-            print('No extension specified. You must specify at least one extension')
+        if not options.extensions and not options.extensions_list:
+            print('No extension specified. You must specify at least one extension or try using default extension list.')
             exit(0)
 
+        if not options.extensions and options.extensions_list:
+            options.extensions = "html,php,tar.gz,txt,xml,zip,jpg,png,jpeg"
```

Comment:
Read extensions from a FILE rather than hardcoding them, see https://github.com/maurosoria/dirsearch/blob/26a9e240bd8de93108f9ac1c400ea49f8634e4fa/lib/core/ArgumentParser.py#L36-L59